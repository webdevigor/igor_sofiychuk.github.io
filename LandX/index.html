<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <link rel="stylesheet" href="build/css/bootstrap.min.css">
    <link rel="stylesheet" href="build/css/slick-theme.css">
    <link rel="stylesheet" href="build/css/slick.css">
    <link rel="stylesheet" href="build/css/main.min.css">
    <title>LandX</title>
</head>

<body>
    <header class="header">

        <div class="container">
            <div class="row">
                <div class="col">
                    <div class="wrap-logo">
                        <div class="logo">
                            <a href="#!">Land<span>X</span></a>
                        </div>
                        <ul class="head-social">
                            <li class="head-social-item">
                                <a href="#!">
                                    <img src="build/img/facebook.svg" alt="facebook">
                                </a>
                            </li>
                            <li class="head-social-item">
                                <a href="#!">
                                    <img src="build/img/twitter.svg" alt="twitter">
                                </a>
                            </li>
                            <li class="head-social-item">
                                <a href="#!">
                                    <img src="build/img/linkedin-logo.svg" alt="linkedin">
                                </a>
                            </li>
                        </ul>
                    </div>
                </div>
            </div>
        </div>

        <div class="container">
            <div class="row">
                <div class="col">
                    <div class="wrap-title">
                        <h1 class="title"> Get the Best <span>Solution</span> for Your Business </h1>
                        <p class="subtitle">
                            Accelerator photo sharing business school drop out ramen
                            hustle crush it revenue traction platforms.
                            Coworking viral landing page user base.
                        </p>
                    </div>
                </div>
            </div>
        </div>

        <div class="container">
            <div class="row">
                <div class="col">
                    <div class="wrap-form">
                        <form class="form">
                            <input type="name" value="Your Name">
                            <input type="email" value="Your E-Mail">
                            <input type="email" value="Phone Number">
                            <input type="submit" class="btn-started" value="GET STARTED">

                        </form>
                        <div class="wrap-arrow">
                            <span>Register Now and Get FREE Trial</span>

                            <div class="arrow">
                                <img src="build/img/arrow-top.png" alt="arrow-top">
                            </div>

                        </div>
                    </div>
                </div>
            </div>
        </div>

    </header>

    <section class="s-explore">
        <div class="container">
            <div class="row">
                <div class="col-lg-6 col-md-12 col-sm-12">
                    <div class="wrap-i">
                        <img src="build/img/browser.png" alt="browser">
                    </div>
                </div>

                <div class="col-lg-6 col-md-12 col-sm-12">
                    <div class="wrap-block">
                        <h3 class="title">Explore the awesomeness</h3>
                        <p class="subtitle">
                            Lorem ipsum dolor sit amet,
                            consectetur adipisicing elit,
                            sed do eiusmod tempor incididunt ut
                            labore et dolore magna aliqua.
                            Ut enim ad minim veniam,
                            quis nostrud exercitation ullamco laboris nisi ut.
                        </p>

                        <ul class="block-list-items">
                            <li class="list-item">
                                <div class="wrap-img">
                                    <img src="build/img/costomise.png" alt="Easy to Customize">
                                </div>
                                <h5>Easy to Customize</h5>
                                <p>Lorem lean startup ipsum product market fit customer development
                                    acquihire technical cofounder.</p>
                            </li>
                            <li class="list-item">
                                <img src="build/img/costomise-2.png" alt="Quality Design">
                                <h5>Quality Design</h5>
                                <p>Lorem lean startup ipsum product market fit customer development
                                    acquihire technical cofounder.</p>
                            </li>
                        </ul>
                    </div>
                </div>
            </div>
        </div>
    </section>

    <section class="describe s-explore">
        <div class="container">
            <div class="row">
                <div class="col-lg-6 col-md-12 col-sm-12">
                    <div class="wrap-block">
                        <h3 class="title">Describe more about product</h3>
                        <p class="subtitle">
                            Lorem ipsum dolor sit amet,
                            consectetur adipisicing elit,
                            sed do eiusmod tempor incididunt ut
                            labore et dolore magna aliqua.
                            Ut enim ad minim veniam,
                            quis nostrud exercitation ullamco laboris nisi ut.
                        </p>

                        <ul class="block-list-items">
                            <li class="list-item">
                                <div class="wrap-img">
                                    <img src="build/img/costomise-3.png" alt="Easy to Customize">
                                </div>
                                <h5>Good in Performance</h5>
                                <p>Lorem lean startup ipsum product market fit customer development
                                    acquihire technical cofounder.</p>
                            </li>
                            <li class="list-item">
                                <img src="build/img/costomise-4.png" alt="Quality Design">
                                <h5>Video Background</h5>
                                <p>Lorem lean startup ipsum product market fit customer development
                                    acquihire technical cofounder.</p>
                            </li>
                        </ul>
                    </div>
                </div>
                <div class="col-lg-6 col-md-12 col-sm-12">
                    <div class="wrap-i">
                        <img src="build/img/browser.png" alt="browser">
                    </div>
                </div>
            </div>
        </div>
    </section>

    <section class="features">
        <div class="container">

            <div class="row">
                <div class="col">

                    <h3 class="title">LandX Features</h3>
                    <p class="subtitle">Term sheet convertible note colluding bootstrapping.</p>

                </div>
            </div>

            <div class="row">
                <div class="col-lg-4 col-md-6 col-sm-12">

                    <div class="wrap-features-cart">

                        <div class="wrap-features-img">
                            <img src="build/img/features-1.png" alt="features-1">
                        </div>
                        <h4 class="caption">Responsive Design</h4>
                        <p class="desc">Lorem lean startup ipsum product
                            market fit
                            customer development acquihire technical cofounder.
                            User engagement.</p>

                    </div>
                </div>
                <div class="col-lg-4 col-md-6 col-sm-12">

                    <div class="wrap-features-cart">

                        <div class="wrap-features-img">
                            <img src="build/img/features-2.png" alt="features-2">
                        </div>
                        <h4 class="caption">1000+ Font Icons</h4>
                        <p class="desc">Lorem lean startup ipsum product
                            market fit
                            customer development acquihire technical cofounder.
                            User engagement.</p>

                    </div>
                </div>
                <div class="col-lg-4 col-md-6 col-sm-12">

                    <div class="wrap-features-cart">

                        <div class="wrap-features-img">
                            <img src="build/img/features-3.png" alt="">
                        </div>
                        <h4 class="caption">Built with B ootstrap 4.1</h4>
                        <p class="desc">Lorem lean startup ipsum product
                            market fit
                            customer development acquihire technical cofounder.
                            User engagement.</p>

                    </div>
                </div>

            </div>

            <div class="row">
                <div class="col-lg-4 col-md-6 col-sm-12">

                    <div class="wrap-features-cart">
                        <div class="wrap-features-img">
                            <img src="build/img/features-4.png" alt="features-4">
                        </div>
                        <h4 class="caption">Lots of Variations</h4>
                        <p class="desc">
                            Lorem lean startup ipsum product
                            market fit
                            customer development acquihire technical cofounder.
                            User engagement.
                        </p>

                    </div>
                </div>
                <div class="col-lg-4 col-md-6 col-sm-12">

                    <div class="wrap-features-cart">

                        <div class="wrap-features-img">
                            <img src="build/img/features-5.png" alt="features-5">
                        </div>
                        <h4 class="caption">Valid Markup</h4>
                        <p class="desc">Lorem lean startup ipsum product
                            market fit
                            customer development acquihire technical cofounder.
                            User engagement.</p>

                    </div>
                </div>
                <div class="col-lg-4 col-md-6 col-sm-12">

                    <div class="wrap-features-cart">

                        <div class="wrap-features-img">
                            <img src="build/img/features-6.png" alt="features-6">
                        </div>
                        <h4 class="caption"> Easy to Customize</h4>
                        <p class="desc">Lorem lean startup ipsum product
                            market fit
                            customer development acquihire technical cofounder.
                            User engagement.</p>

                    </div>
                </div>


            </div>

        </div>
    </section>

    <section class="pricing">
        <div class="container">

            <div class="row">
                <div class="col">
                    <h3 class="title">Pricing Table</h3>
                    <p class="subtitle">Term sheet convertible note colluding bootstrapping.</p>
                </div>
            </div>

            <div class="row">
                <div class="col-lg-4 col-md-6 col-sm-12">
                    <div class="wrap-top ">
                        <h4 class="caption">Basic</h4>
                        <p class="price">From<span class="b-price">$99</span>per month</p>
                    </div>
                    <ul class="price-list">
                        <li class="price-list-item">Total User <span>15</span></li>
                        <li class="price-list-item">Unlimited Styles <span>&#10004;</span></li>
                        <li class="price-list-item">Advance Protection <span>&#10004;</span></li>
                        <li class="price-list-item"> Cloud Storage<span>&#10008;</span></li>
                        <li class="price-list-item">24/7 Customer Service <span>&#10008;</span></li>
                        <li class="price-list-item">Backup Service <span>&#10008;</span></li>
                        <li class="price-list-item spec">
                            $ 99 Per Month
                            <a href="#!">
                                <button class="btn-sing-up">SING UP</button>
                            </a>
                        </li>
                    </ul>
                </div>

                <div class="col-lg-4 col-md-6 col-sm-12">
                    <div class="wrap-top premium">
                        <h4 class="caption">Premium</h4>
                        <p class="price ">From<span class="p-price">$199</span>per month</p>
                    </div>
                    <ul class="price-list">
                        <li class="price-list-item">Total User <span>15</span></li>
                        <li class="price-list-item">Unlimited Styles <span>&#10004;</span></li>
                        <li class="price-list-item">Advance Protection <span>&#10004;</span></li>
                        <li class="price-list-item"> Cloud Storage<span>&#10004;</span></li>
                        <li class="price-list-item">24/7 Customer Service <span>&#10004;</span></li>
                        <li class="price-list-item">Backup Service <span>&#10008;</span></li>
                        <li class="price-list-item spec">
                            $99 Per Month
                            <a href="#!">
                                <button class="btn-sing-up prem">SING UP</button>
                            </a>
                        </li>
                    </ul>
                </div>
                <div class="col-lg-4 col-md-6 col-sm-12">
                    <div class="wrap-top">
                        <h4 class="caption">Ultimate</h4>
                        <p class="price">From<span class="b-price">$299</span>per month</p>
                    </div>
                    <ul class="price-list">
                        <li class="price-list-item">Total User <span>15</span></li>
                        <li class="price-list-item">Unlimited Styles <span>&#10004;</span></li>
                        <li class="price-list-item">Advance Protection <span>&#10004;</span></li>
                        <li class="price-list-item">Cloud Storage<span>&#10004;;</span></li>
                        <li class="price-list-item">24/7 Customer Service <span>&#10004;</span></li>
                        <li class="price-list-item">Backup Service <span>&#10004;</span></li>
                        <li class="price-list-item spec">
                            $99 Per Month
                            <a href="#!">
                                <button class="btn-sing-up ">SING UP</button>
                            </a>
                        </li>
                    </ul>
                </div>
            </div>

        </div>
    </section>

    <section class="video">
        <div class="container">
            <div class="row">
                <div class="col">
                    <h3 class="title">Watch Video</h3>
                    <p class="subtitle">Term sheet convertible note colluding bootstrapping.</p>
                </div>
            </div>
            <div class="row">
                <div class="col">
                    <div class="relation if-video">
                        <div class="relation-ratio"></div>
                        <iframe class="relation-content" src="https://www.youtube.com/embed/MwD_tZGc7gA"
                            allowfullscreen></iframe>
                    </div>
                    <ul class="block-list">
                        <li class="item-list"> <img src="build/img/users.png" alt="users"> Trusted by 100+ users</li>
                        <li class="item-list"><img src="build/img/chat-support.png" alt="chat-support">Video
                            Documentation</li>
                        <li class="item-list"><img src="build/img/video-documentation.png"
                                alt="video-documentation">24/7 Chat Support</li>
                    </ul>
                </div>
            </div>
        </div>
    </section>
    <section class="product">
        <div class="container">
            <div class="row">
                <div class="col">
                    <h3 class="title">Product Screenshots</h3>
                    <p class="subtitle">Term sheet convertible note colluding bootstrapping.</p>
                </div>
            </div>
            <div class="row">
                <div class="col">
                    <div class="slider-sl">

                        <div class="sider-img">
                            <img src="build/img/screenshot-1.jpg" alt="screenshot-1">
                        </div>

                        <div class="sider-img">
                            <img src="build/img/screenshot-2.jpg" alt="screenshot-2">
                        </div>
                        <div class="sider-img">
                            <img src="build/img/screenshot-3.jpg" alt="screenshot-3">
                        </div>

                    </div>

                </div>
            </div>
        </div>
    </section>

    <section class="feedback">
        <div class="container">
            <div class="row">
                <div class="col">
                    <h3 class="title">Trusted by Thousands</h3>
                    <p class="subtitle">Term sheet convertible note colluding bootstrapping.</p>
                </div>
            </div>
            <div class="row">
                <div class="col">
                    <ul class="list-img">
                        <li class="item"><img src="build/img/tbyt-1.jpg" alt="tbyt-1"></li>
                        <li class="item"><img src="build/img/tbyt-2.jpg" alt="tbyt-2"></li>
                        <li class="item"><img src="build/img/tbyt-3.jpg" alt="tbyt-3"></li>
                        <li class="item"><img src="build/img/tbyt-4.jpg" alt="tbyt-4"></li>
                    </ul>
                </div>
            </div>
            <div class="row">
                <div class="col-lg-4 col-md-6 col-sm-12">
                    <div class="wrap">

                        <div class="wrap-img">
                            <img src="build/img/quote.png" alt="quote">
                        </div>

                        <p class="p"> Lorem lean startup ipsum product market
                            fit customer development acquihire technical cofounder.
                            User engagement A/B testing shrink a market venture capital pitch deck.
                            Social bookmarking group buying crowded market pivot onboarding.
                        </p>

                    </div>
                    <div class="bottom-text">
                        <p class="sub">Willeke Hoekstra</p>
                        <p class="sub-2">AbZ Network</p>
                    </div>
                </div>

                <div class="col-lg-4 col-md-6 col-sm-12">
                    <div class="wrap">

                        <div class="wrap-img">
                            <img src="build/img/quote.png" alt="quote">
                        </div>

                        <p class="p"> Lorem lean startup ipsum product market
                            fit customer development acquihire technical cofounder.
                            User engagement A/B testing shrink a market venture capital pitch deck.
                            Social bookmarking group buying crowded market pivot onboarding.
                        </p>

                    </div>
                    <div class="bottom-text">
                        <p class="sub">Răducu Neamțu</p>
                        <p class="sub-2">AbZ Network</p>
                    </div>
                </div>

                <div class="col-lg-4 col-md-6 col-sm-12">
                    <div class="wrap">

                        <div class="wrap-img">
                            <img src="build/img/quote.png" alt="quote">
                        </div>

                        <p class="p"> Lorem lean startup ipsum product market
                            fit customer development acquihire technical cofounder.
                            User engagement A/B testing shrink a market venture capital pitch deck.
                            Social bookmarking group buying crowded market pivot onboarding.
                        </p>

                    </div>
                    <div class="bottom-text">
                        <p class="sub">Markel Díaz</p>
                        <p class="sub-2">AbZ Network</p>
                    </div>
                </div>
            </div>
        </div>
    </section>

    <footer class="footer-top">
        <div class="container">
            <div class="row">
                <div class="col">
                    <p class="headline">
                        We Are Ready to Help You
                    </p>
                    <h3 class="caption">Get the Best Solution for Your Business</h3>
                    <a class="btn-started-2" href="#!"> GET STARTED <img src="build/img/arrow-left.png"
                            alt="arrow-left"></a>
                </div>
            </div>
        </div>
    </footer>

    <footer class="footer-center">
        <div class="container">
            <div class="row">
                <div class="col">
                    <div class="wrap-title">
                        <h4 class="caption">Have any question or need help?</h4>
                        <button class="btn-mail">CONTACT US NOW <img src="build/img/mail.svg" alt="mail"></button>
                    </div>
                </div>
            </div>
        </div>
    </footer>

    <footer class="footer-bottom">
        <div class="container">
            <div class="row">
                <div class="col">
                    <div class="footer-logo">
                        <a href="#!">Land<span>X</span></a>
                    </div>
                    <h5 class="subject">©2014 LandX Template LLC.</h5>
                    <ul class="footer-social">
                        <li class="footer-social-item">
                            <a href="#!">
                                <img src="build/img/facebook-footer.svg" alt="facebook-footer">
                            </a>
                        </li>
                        <li class="footer-social-item">
                            <a href="#!">
                                <img src="build/img/twitter-footer.svg" alt="twitter-footer">
                            </a>
                        </li>
                        <li class="footer-social-item">
                            <a href="#!">
                                <img src="build/img/pinterest-footer.svg" alt="pinterest-footer">
                            </a>
                        </li>
                        <li class="footer-social-item">
                            <a href="#!">
                                <img src="build/img/google-footer.svg" alt="google-footer">
                            </a>
                        </li>
                        <li class="footer-social-item">
                            <a href="#!">
                                <img src="build/img/instagram-footer.svg" alt="instagram-footer">
                            </a>
                        </li>
                        <li class="footer-social-item">
                            <a href="#!">
                                <img src="build/img/linkedin-footer.svg" alt="linkedin-footer">
                            </a>
                        </li>
                    </ul>
                </div>
            </div>
        </div>
    </footer>

    <script src="build/js/main.min.js"></script>
    <script src="build/js/slick.min.js"></script>

</body>

</html>