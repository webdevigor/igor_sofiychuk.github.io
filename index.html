<!DOCTYPE html>
<html lang="ru">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700" rel="stylesheet">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <link rel="stylesheet" href="css/style.css">
    <title>Calc</title>
</head>
<body>
    <div class="calc">
        <div class="tabs">
            
            <div class="tab-left activ">Лендинг</div>
            <div class="tab-right">Многостраничник</div>
        </div>
        <div class="blocks" id="blocks-block">
           <div class="blocks-title">Количество блоков</div> 
           <input type="number" class="counter" id="counter-block">
        </div>
        <div class="blocks" id="pages-block">
            <div class="blocks-title">Количество страниц</div> 
            <input type="number" class="counter" id="counter-pages">
        </div>
        <div class="blocks">
                <div class="blocks-title">Сколько часов на проект</div> 
                <input type="number" class="counter" id="counter-hours">
        </div>
        <div class="blocks">
                <div class="blocks-title">Стоимость вашего часа</div> 
                <input type="number" class="counter" id="counter-rate">
        </div>
        <div class="addiction">
            <div class="addiction-title">Дополнительно </div>
            <div class="addiction-block">
                <label for="changes-check">
                        <input class="checkbox" type="checkbox" id="changes-check">
                        Возможность редактировать текст на сайте
                </label>
                
            </div>
            <div class="addiction-block">
                <label for="changes-cms">
                        <input class="checkbox" type="checkbox" id="changes-cms">
                        Установка на CMS
                </label>
                    
             </div>
        </div>
        <div class="total">
            Общая стоимость
            <input value="5000" type="text" class="total-count" id="total-value">
        </div>
    </div>

    <script src="script.js"></script>
</body>
</html>