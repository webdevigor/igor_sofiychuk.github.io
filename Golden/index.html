<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css">
    <link rel="stylesheet" href="css/style.css">
    <link rel="stylesheet" href="fonts/Wisdom Script AJ.otf">
    <link href="https://fonts.googleapis.com/css?family=Allura|Montserrat|Roboto|Roboto+Slab" rel="stylesheet">
    <title>Golden</title>
</head>

<body>
    <header class="">
        <div class="container">
            <div class="row">
                <div class="col-md-12">
                    <div class="logo">
                        <a href="#">Golden</a>
                    </div>
                </div>
                <div class="col">
                    <ul class="nav-panel nav justify-content-end">
                        <li class="nav-item">
                            <a href="#">home</a>
                        </li>
                        <li class="nav-item">
                            <a href="#">services</a>
                        </li>
                        <li class="nav-item">
                            <a href="#">portfolio</a>
                        </li>
                        <li class="nav-item">
                            <a href="#">about</a>
                        </li>
                        <li class="nav-item">
                            <a href="#">contact</a>
                        </li>
                    </ul>
                </div>

            </div>


            <div class="row text-center">
                <div class="col">
                    <h3 class="text-header">Welcome To Our Studio</h3>
                </div>
            </div>

            <div class="row text-center">
                <div class="col">
                    <h1 class="text-header-2">Itâ€™s nice to Meet you</h1>
                </div>

            </div>

            <div class="row">
                <div class="col text-center">
                    <button class="header-btn btn">
                        Tell Me More
                    </button>
                </div>
            </div>
        </div>
    </header>

    <section>
        <div class="container">
            <div class="row">
                <div class="col text-center section-title">

                    <h3>services</h3>
                    <p>
                        Proint iaculis purus consequat sem cure.
                    </p>
                </div>
            </div>
            <div class="row">
                <div class="col text-center section-img">
                    <img class="section-img-icon" src="img/svg/Icon-Basket@1,25x.svg" alt="#" style="width: 40px; height: 40px;">
                    <img class="section-img-circle" src="img/svg/Circle-hover-icon-Basket.svg" alt="#">

                    <h3>E-Commerce</h3>
                    <p>
                        Proint iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.
                    </p>
                </div>
                <div class="col text-center section-img">
                    <img class="section-img-icon" src="img/svg/Icon-Laptop@1,25x.svg" alt="#" style="width: 40px; height: 40px;">
                    <img class="section-img-circle" src="img/svg/Circle-hover-icon-Basket.svg" alt="#">

                    <h3>Responsive Web</h3>
                    <p>
                        Proint iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
                </div>
                <div class="col text-center section-img">
                    <img class="section-img-icon" src="img/svg/Icon-Locked@1,25x.svg" alt="#" style="width: 40px; height: 40px;">
                    <img class="section-img-circle" src="img/svg/Circle-hover-icon-Basket.svg" alt="#">
                    <h3>Web Security</h3>
                    <p>
                        Proint iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.
                    </p>
                </div>
            </div>
        </div>
    </section>

    <section>
        <div class="container">

            <div class="row">
                <div class="col text-center section-title">
                    <h3>our portfolio</h3>
                    <p>Proint iaculis purus consequat sem cure.</p>
                </div>
            </div>

            <div class="row justify-content-center img-cont">
                <div class="col-lg-4 col-md-6 col-sm-8">
                    <img class="img-thumbnail" src="img/ebony-lvory.jpg" alt="#">
                    <h4 class="text-center img-title">Ebony and Ivory</h4>
                    <p class="text-center img-title-2">Branding</p>
                </div>
                <div class="col-lg-4 col-md-6 col-sm-8">
                    <img class="img-thumbnail" src="img/smart-stationary.jpg" alt="#">
                    <h4 class="text-center img-title">Smart Stationary</h4>
                    <p class="text-center img-title-2">Print Desing</p>
                </div>
                <div class="col-lg-4 col-md-6 col-sm-8">
                    <img class="img-thumbnail" src="img/clever-poster.jpg" alt="#">
                    <h4 class="text-center img-title">Clever Poster</h4>
                    <p class="text-center img-title-2">Print Desing </p>
                </div>
            </div>
            <div class="row justify-content-center img-cont">
                <div class="col-lg-4 col-md-6 col-sm-8">
                    <img class="img-thumbnail" src="img/vinyl-record.jpg" alt="#">
                    <h4 class="text-center img-title">Vinyl Record</h4>
                    <p class="text-center img-title-2">Product Mock-Up</p>
                </div>
                <div class="col-lg-4 col-md-6 col-sm-8">
                    <img class="img-thumbnail" src="img/reehouse-emplate.jpg" alt="#">
                    <h4 class="text-center img-title">Treehouse Template</h4>
                    <p class="text-center img-title-2">Web Desing</p>
                </div>
                <div class="col-lg-4 col-md-6 col-sm-8">
                    <img class="img-thumbnail" src="img/burned-logo.jpg" alt="#">
                    <h4 class="text-center img-title">Burned-Logo</h4>
                    <p class="text-center img-title-2">Branding</p>
                </div>
            </div>
        </div>
    </section>


    <section class="about">
        <div class="container">
            <div class="row">
                <div class="col text-center section-title">
                    <h3>
                        aboot us
                    </h3>
                    <p>
                        Proint iaculis purus consequat sem cure.
                    </p>
                </div>
            </div>
            <div class="row ">

                <div class="col-lg-4 col-md-6 col-sm-8 offset-lg-1">
                    <h4 class="text-right">July 2010</h4>
                    <h4 class="text-right">Our Humble Beginnings</h4>
                    <p class="text-right"> Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere
                        odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit.</p>
                </div>
                <div class="col-4 col-md-6 col-sm-4 block-about">
                    <img src="img/yule-2010.jpg" alt="#">
                </div>

                <div class="col-4 col-sm-5 text-right offset-lg-2 block-about">
                    <img class="img-about" src="img/january-2011.jpg" alt="#">
                </div>
                <div class="col-lg-4 col-md-6 col-sm-7 ">
                    <h4 class="about-text-right">January 2011</h4>
                    <h4 class="about-text-right">Facing Startup Battles</h4>
                    <p class="about-text-right">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere
                        odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit
                        aenean rhoncus posuere odio in tincidunt.</p>
                </div>

                <div class="col-lg-4 col-md-6 col-sm-8 offset-lg-1">
                    <h4 class="text-right">December 2012</h4>
                    <h4 class="text-right">Enter The Dark Days</h4>
                    <p class="text-right">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere
                        odio in tincidunt. Proin iaculis purus consequat sem cure digni.</p>
                </div>
                <div class="col-6 col-sm-4 block-about">
                    <img src="img/december-2012.jpg" alt="#">
                </div>

                <div class="col-5 col-sm-5 text-right offset-lg-2 block-about">
                    <img class="img-about" src="img/februar-2014.jpg" alt="#">
                </div>
                <div class="col-lg-4 col-md-6 col-sm-7">
                    <h4 class="about-text-right">february 2014</h4>
                    <h4 class="about-text-right">Our Triumph</h4>
                    <p class="about-text-right">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere
                        odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit
                        aenean.
                    </p>
                </div>

                <div class="col-lg-4 col-md-8 col-sm-8 offset-lg-5 offset-md-5 offset-sm-4">
                    <div class="img-last-about">
                        <h4>our
                            <br> story
                            <br> continues
                            <br>...</h4>
                    </div>
                </div>
            </div>
        </div>
    </section>

    <section>
        <div class="container">
            <div class="row">
                <div class="col text-center section-title">
                    <h3>our amazing team</h3>
                    <p>Proin iaculis purus consequat sem cure.</p>
                </div>
            </div>
            <div class="row">
                <div class="col-lg-4 col-md-4 col-sm-4 text-center cirkle-text">
                    <div class="cirkle ">

                    </div>
                    <h4>Kimberly Thompson</h4>
                    <p>Marketer</p>
                    <div class="cirkle-img-svg">
                        <a href="#">
                            <img src="img/svg/social-twitter-circular-button.svg" alt="#">
                        </a>
                        <a href="#">
                            <img src="img/svg/social-facebook-circular-button.svg" alt="#">

                        </a>
                        <a href="#">
                            <img src="img/svg/if_Pinterest_194917.svg" alt="#">
                        </a>
                    </div>
                </div>
                <div class="col-lg-4 col-md-4 col-sm-4 text-center cirkle-text">
                    <div class="cirkle">

                    </div>
                    <h4>Rico Massimo</h4>
                    <p>Coder</p>
                    <div class="cirkle-img-svg">
                        <a href="#">
                            <img src="img/svg/social-twitter-circular-button.svg" alt="#">
                        </a>
                        <a href="#">
                            <img src="img/svg/social-facebook-circular-button.svg" alt="#">

                        </a>
                        <a href="#">
                            <img src="img/svg/if_Google__194926.svg" alt="#">
                        </a>
                    </div>
                </div>
                <div class="col-lg-4 col-md-4 col-sm-4 text-center cirkle-text">
                    <div class="cirkle">

                    </div>
                    <h4>Uku Mason</h4>
                    <p>Graphic Designer</p>
                    <div class="cirkle-img-svg">
                        <a href="#">
                            <img src="img/svg/social-facebook-circular-button.svg" alt="#">

                        </a>
                        <a href="#">
                            <img src="img/svg/if_Pinterest_194917.svg" alt="#">
                        </a>
                    </div>
                </div>
            </div>
            <div class="row">
                <div class="col text-center cirkle-bot-text">
                    <p>
                        Proin iaculis purus consequat sem cure digni ssim donec porttitora entum suscipit
                        <br> aenean rhoncus posuere odio in tincidunt proin iaculis.
                    </p>
                </div>
            </div>
        </div>
    </section>

    <section>
        <div class="container">
            <div class="row mt-2">
                <div class="col-3">
                    <img src="img/Envato.png" alt="#">
                </div>
                <div class="col-3">
                    <img src="img/WordPress.jpg" alt="#">
                </div>
                <div class="col-3">
                    <img src="img/Tuts+.jpg" alt="#">
                </div>
                <div class="col-3">
                    <img src="img/Microlancer copy.jpg" alt="#">
                </div>
            </div>
        </div>
    </section>

    <footer class="footer-bi">
        <div class="container">
            <div class="row">
                <div class="col text-center section-title text-white">
                    <h3>Contact US</h3>
                    <p>Proin iaculis purus consequat sem cure</p>
                </div>
            </div>

            <form>
                <div class="row">
                    <div class="col-4 offset-2">
                        <input type="text" class="form-control" placeholder="YOUR NAME*">
                        <input type="email" class="form-control" placeholder="YOUR E-MAIL*">
                        <input type="text" class="form-control" placeholder="SUBJECT">
                    </div>

                    <div class="col-4">
                        <textarea class="form-control" rows="5">YOUR MESSAGE</textarea>
                    </div>
                </div>
                <div class="row">
                    <div class="button-send col-3 offset-5 text-ceter">
                        <button class="btn btn-warning text-white">SEND MASSEGE</button>
                    </div>
                </div>
            </form>
        </div>
    </footer>

    <footer>
        <div class="container footer-sicial">
            <div class="row">
                <div class="col-4 text-center">
                    <p>
                        Â© Copyright 2014 FreebiesXpress.com
                    </p>

                </div>
                <div class="col-4 text-center">
                    <img src="img/svg/social-twitter-circular-button.svg" alt="#">

                    <img src="img/svg/social-facebook-circular-button.svg" alt="#">

                    <img src="img/svg/if_Pinterest_194917.svg" alt="#">

                    <img src="img/svg/if_Google__194926.svg" alt="#">
                </div>
            </div>
        </div>

    </footer>



</body>

</html>